static/app/components/forms/controls/rangeSlider.tsx

```diff
@@ -6,7 +6,7 @@ import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 import {defined} from 'sentry/utils';
 
-type Props = {
+export type Props = {
```

Comment:
Give props and label a more descriptive name when exporting them

Reply:
I would think about going a step further too and moving the exported styled components to their own files since they are being used as components. (we generally only export one component per file).

e.g. rangeSlider.tsx --> rangeSlider/index.tsx, label --> rangerSlider/sliderLabel.tsx, etc